Lesson-1/assignment/yours.sol

```diff
@@ -1 +1,51 @@
 /*作业请提交在这个目录下*/
+pragma solidity ^0.4.14;
+
+contract Payroll {
+    uint constant payDuration = 10 seconds;
+
+    address owner;
+    uint salary;
+    address employee;
+    uint lastPayday;
+
+    function Payroll() {
+        owner = msg.sender;
+    }
+    
+    function updateEmployeeAddress(address e) {
+        require(msg.sender == owner);
+        
+        employee = e;
+        lastPayday = now;
+    }
+
+    function updateEmployeeSalary(uint s) {
+        require(msg.sender == owner);
+        
```

Comment:
Need to pay the rest due to the employee first before modifying, if the salary is not paid yet.

Reply:
Thanks updated in lesson two homework with partialPay func.